apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-button/index.ts

```diff
@@ -44,7 +44,11 @@ domReady( () => {
 
 		// Wrap 'Launch' button link to control launch flow.
 		const launchButton = document.createElement( 'a' );
-		launchButton.href = launchUrl;
+		// This was previously launchUrl but setting it to javascript: void(0)
+		// to prevent users from doing through site launch when they are
+		// assigned to focused launch. This can happen if they hit CMD+Click
+		// or Right Click + Open In New Tab.
+		launchButton.href = 'javascript: void(0);';
```

Comment:
Though unlikely, this *might* be one reason users assigned to focused launch still end up with site launch.